docs/source/operations/projections/peirce_q.rst

```diff
@@ -35,6 +74,40 @@ Parameters
 
 .. include:: ../options/lon_0.rst
 
+.. option:: +type=square/diamond/horizontal/vertical/nhemisphere/shemisphere
+
+    .. versionadded:: 8.2.1
+
+    *Defaults to diamond.*
+
+    Indicates the type of transformation applied to the southern hemisphere: ``square`` and
+    ``diamond`` represent the traditional quincuncial form suggested by Peirce with the
+    southern hemisphere divided into 4 triangles and reflected outward from the northern
+    hemisphere. The ``square`` type is rotated by 45 degrees to produce the conventional
+    square presentation. The origin lies at the centre of the square or diamond.
+
+    By contrast, the ``horizontal`` and ``vertical`` forms reflect the southern hemisphere
+    laterally across the x or y axis respectively to produce a rectangular form. The origin
+    lies at the centre of the rectangle.
+
+    The other two types, ``nhemisphere`` and ``shemisphere``, discard latitudes of less
+    than 0 or more than 0, respectively, to allow single hemispheres to be selected.
+    The origin lies at the centre of the square or diamond.
+
+.. option:: +o_scrollx=<value>
```

Comment:
Parameters should't start with o_  . o_ is a prefix used by +proj=ob_tran to know that the parameters must be forwarded to the projection identified by +o_proj without the o_ prefix

Reply:
OK, have changed o_scrollx to scrollx in code+docs, but see note above that o_scrollx does not get passed as scrollx to peirce_q via ob_tran